src/main/resources/freemarker/data-custodian-approval.html

```diff
@@ -22,7 +22,7 @@
         <tr>
             <td id="content" style="padding: 0px 15px; font-family: 'Montserrat', sans-serif; font-size: 16px; color: #1F3B50; text-align: justify; line-height: 25px;">
                 <p>
-                    ${darCode} submitted by <a style="text-decoration: none; font-family: 'Montserrat', sans-serif; color: #00609F;">${researcherEmail}</a> was approved by the DAC for the following datasets:
+                    ${darCode} submitted by <a style="text-decoration: none; font-family: 'Montserrat', sans-serif; color: #00609F;">${researcherEmail}</a> was ${radarText}approved by the DAC for the following datasets:
```

Comment:
Minor observation. In the various email constructions, `radarText` is suffixed with a space. I think you probably don't need that and add the space here in the email. Double spaces in html should be rendered as a single space.